mvs_eland_tool/mvs_eland_tool.py

```diff
@@ -71,10 +71,11 @@ def main(**kwargs):
     #    # todo: is user input completely used?
     #    dict_values = data_input.load_json(user_input["path_input_file"])
 
-    if not user_input["path_input_file"].endswith("json"):
+    if user_input["input_type"] == "csv":
```

Comment:
```suggestion
    if user_input["input_type"] == CSV_EXT:
```

You might need to import it above in the file